EC_process/onboarding.md

```diff
@@ -0,0 +1,10 @@
+# Onboarding (and offboarding) of Executive Council Members
+
+The official term of office for new Executive Council (EC) Members is February 1. We ask that new Members attend the regularly scheduled EC meeting in January prior to taking office to learn how meetings operate.
+
+* Permissions and access to the following resources should be modified in late December or early January:
+  * Google Drive
+  * GitHub (organization, teams)
+  * Listserv
```

Comment:
This is the mailing list we use, right? Maybe avoid the technical term (jargon?) 'listserv' here?